From 74b5c35a96c13d12c16e2dd0e8d97f37d8765807 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 9 Sep 2020 15:17:09 +0800
Subject: [PATCH] s32g274aevb: add secure boot support

Issue: LINUXEXEC-9576

Add related config options to enable the secure boot support
for s32g274aevb board.

(LOCAL REV: NOT UPSTREAM) -- WRLinux specific

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 configs/s32g274aevb_defconfig | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/configs/s32g274aevb_defconfig b/configs/s32g274aevb_defconfig
index 8f83bcfa54..0f7f0320e8 100644
--- a/configs/s32g274aevb_defconfig
+++ b/configs/s32g274aevb_defconfig
@@ -1,7 +1,11 @@
 CONFIG_ARM=y
 CONFIG_ARM_SMCCC=y
 CONFIG_ARCH_S32=y
-CONFIG_SYS_TEXT_BASE=0x34020000
+CONFIG_SYS_TEXT_BASE=0x34080000
+CONFIG_HSE_SECBOOT=y
+CONFIG_CMD_HSE_ADVANCED_SECBOOT=y
+CONFIG_FIT_SIGNATURE=y
+CONFIG_ENV_IS_NOWHERE=y
 CONFIG_TARGET_S32G274AEVB=y
 CONFIG_S32_LPDDR4=y
 CONFIG_SYS_DDR_SIZE=0x20000000
-- 
2.17.1

